config/main.py

```diff
@@ -199,6 +199,23 @@ def validate_namespace(namespace):
     else:
         return False
 
+"""In case of Multi-Asic platform, Each ASIC will have a host name and we call it internal hosts.
+   So we loop through the databases in different namespaces and get the hostname
+"""
```

Comment:
Prefer to reserve`"""` for docstrings and use `#` for comments

Reply:
Done.